cmd/ak/cmd/vars/set.go

```diff
@@ -32,16 +32,35 @@ var setCmd = common.StandardCommand(&cobra.Command{
 		}
 
 		var value string
-		if len(args) == 2 {
-			value = args[1]
+
+		isValueSet := len(args) > 1
+
+		if isValueSet {
+			if args[1] != "-" {
+				value = args[1]
+			} else {
+				const maxVarSize = 1 << 20 // 1MB
+				r := io.LimitReader(os.Stdin, maxVarSize)
+				data, err := io.ReadAll(r)
+				if err != nil {
+					return err
+				}
+				value = string(data)
+			}
 		} else {
-			const maxVarSize = 1 << 20 // 1MB
-			r := io.LimitReader(os.Stdin, maxVarSize)
-			data, err := io.ReadAll(r)
+			// get current var in order not to overwrite it with empty.
```

Comment:
Why? It's perfectly reasonable to want to clear the content of a var without deleting it.